packages/kbn-url-state/index.test.ts

```diff
@@ -28,95 +27,26 @@ describe('useSyncToUrl', () => {
     window.history = {
       ...originalHistory,
       replaceState: jest.fn(),
+      pushState: jest.fn(),
     };
+
+    jest.useFakeTimers();
   });
 
   afterEach(() => {
     window.location = originalLocation;
     window.history = originalHistory;
+    jest.useRealTimers();
   });
 
-  it('should restore the value from the query string on mount', () => {
-    const key = 'testKey';
-    const restoredValue = { test: 'value' };
-    const encodedValue = encode(restoredValue);
-    const restore = jest.fn();
-
-    window.location.search = `?${key}=${encodedValue}`;
-
-    renderHook(() => useSyncToUrl(key, restore));
-
-    expect(restore).toHaveBeenCalledWith(restoredValue);
-  });
-
-  it('should sync the value to the query string', () => {
-    const key = 'testKey';
-    const valueToSerialize = { test: 'value' };
-
-    const { result } = renderHook(() => useSyncToUrl(key, jest.fn()));
-
-    act(() => {
-      result.current(valueToSerialize);
-    });
-
-    expect(window.history.replaceState).toHaveBeenCalledWith(
-      { path: expect.any(String) },
-      '',
-      '/?testKey=%28test%3Avalue%29'
-    );
-  });
-
-  it('should should not alter the location hash', () => {
```

Comment:
Would be great to know the reason for removing these tests because this can surely use many more tests.